<!--
Runs all the Seabotix BTA252 thrusters.
-->

<launch>
    <!-- Thrusters recieve commands and publishes status. -->
    <node 
        pkg="sbBTA252"
        type="sbBTA252"
        name="thrusters">
        <param
			name="numThrust"
			value="1"
		/>
        <param
            name="port"
            value="/dev/ttyS0"
        />
        <param
            name="baud"
            value="115200"
        />
        <param
            name="initTime"
            value="3"
        />
        <param
            name="reconnect"
            value="true"
        />
    </node>
</launch>
